docs/src/modules/brandingTheme.ts

```diff
@@ -189,7 +189,14 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
     spacing: 10,
     typography: {
       fontFamily: ['"IBM Plex Sans"', ...systemFont].join(','),
-      fontFamilyCode: ['"IBM Plex Mono"', ...systemFont].join(','),
+      fontFamilyCode: [
+        '"IBM Plex Mono"',
```

Comment:
On some of the final polishing PRs for the website launch, we discussed using IBM Plex Mono instead of Consolas as the main code typeface. We ended up settling for Consolas which has the upside of being already available on Windows machines, not needing to be downloaded.

I personally really like the IBM Plex Mono/Sans matching though, as they are from the same family. But I also don't have any strong argument to not go Consolas as default and just ditch Plex Mono at all. If the download of Plex Mono fails for some reason Consolas will be the one to render as fallback anyway, so I figure it wouldn't be a problem if it did happen. 

Both read fine to me, so I'm down to go with whichever the team thinks suits better despite my personal taste.

Reply:
Alright, happy with either font, what I want to avoid is https://twitter.com/olivtassinari/status/1437819770896896003. Could you ping the other people involved? It will help hear more voices.

Regarding the font itself, I personally think that the one we use in the docs is a big too small (13px) to read comfortably. I have increased it to 14.

The advantage of `Menlo` is that it's the default font used by VSCode on macOS. It's also the font I see used by React.js, nextjs.org, gatsbyjs.com, Tailwind, getbootstrap.com (I don't have SFMono-Regular installed). Consolas seems almost identical.

So I think that it's about either we want to diverge a bit, be creative, unique (IBM Plex Mono), or we want to rely on the past learnings of the developers to make them feel at home (Menlo/Consolas).

Reply:
Just pushed a commit for using Consolas as the typeface. I'm not feeling like this debate is worth it at this point 😅 We can keep using IBM Plex Mono for the code snippets on the marketing website for a more interesting look while playing safe on the docs with a more common use font.

Regarding the fontSize, both code blocks and inline code are now with 14px, by using the body2 variant. It seems fine.

Reply:
> I'm not feeling like this debate is worth it at this point 😅

Ok, fair enough

> We can keep using IBM Plex Mono for the code snippets on the marketing website for a more interesting look while playing safe on the docs with a more common use font.

I went on using IBM Plex Mono everywhere on mui.com. My assumption was that we should use the same font to create a consistent experience. The latest commits remove the usage of IBM Plex Mono from the marketing website.. Are you ok with this? If you are, I think that we can remove the loading of IBM Plex Mono and consolidate around a single font weight.

Reply:
Oh shoot, I thought that removing from the Markdown Element only affected the docs. But, even though IBM Plex Mono on the marketing pages does add some sugar, for the sake of simplicity, I'm fine with only using one monospace typeface!

I'm on it removing the import of it.

Reply:
Done, the last two commits remove the import and delete the font from the repo.

Reply:
If you would rather keep two different tradeoffs for docs vs. marketing pages, I can retrofit the exercise, and only focus on the small fixes.

Reply:
I think it's fine. It's for sure better to use only one typeface for monospace fonts, simpler and more consistent. There is no strong argument for having one for marketing pages and one for docs 😕 Are you okay with resolving this one?